toolkit/smart-contracts/offchain/src/plutus_script.rs

```diff
@@ -28,27 +29,22 @@ impl std::fmt::Debug for PlutusScript {
 }
 
 impl PlutusScript {
+	/// Constructs a [PlutusScript].
 	pub fn from_cbor(cbor: &[u8], language: Language) -> Self {
 		Self { bytes: cbor.into(), language }
 	}
 
-	pub fn from_ogmios(ogmios_script: OgmiosScript) -> anyhow::Result<Self> {
-		ogmios_script.try_into()
-	}
-
 	/// This function is needed to create [PlutusScript] from scripts in [raw_scripts],
 	/// which are encoded as a cbor byte string containing the cbor of the script
 	/// itself. This function removes this layer of wrapping.
-	pub fn from_wrapped_cbor(
-		plutus_script_raw_cbor: &[u8],
-		language: Language,
-	) -> anyhow::Result<Self> {
+	/// Language for all scrips in [raw_scripts] is [LanguageKind::PlutusV2].
+	pub fn from_raw(plutus_script_raw_cbor: &[u8]) -> anyhow::Result<Self> {
```

Comment:
I don't like this rename. Unwrapping one layer of cbor is vital part of this function.